File: templates/javaee-app/integration-tests/src/test/java/org/sgodden/tom/integration/AbstractIntegrationTest.java
package org.sgodden.tom.integration;

import org.springframework.test.context.ContextConfiguration;
import org.springframework.test.context.testng.AbstractTestNGSpringContextTests;
import org.testng.annotations.BeforeSuite;

import java.util.logging.Level;
import java.util.logging.Logger;

@ContextConfiguration(locations={
        "/org/sgodden/tom/services/impl/beans.xml",
        "/org/sgodden/tom/persistence/beans.xml",
        "/org/sgodden/tom/model/beans.xml"})
public class AbstractIntegrationTest extends AbstractTestNGSpringContextTests {

    @BeforeSuite
    public void beforeSuite() {
        Logger.getLogger("org").setLevel(Level.INFO);
    }

}

File: templates/scala-mongodb/integration-tests/src/test/scala/org/sgodden/tom/integration/AbstractIntegrationTest.scala
package org.sgodden.tom.integration

import org.springframework.test.context.ContextConfiguration
import java.util.logging.{Level, Logger}
import org.testng.annotations.BeforeSuite
import org.springframework.test.context.testng.AbstractTestNGSpringContextTests

@ContextConfiguration (locations = Array (
  "/org/sgodden/tom/services/impl/beans.xml",
  "/org/sgodden/tom/persistence/beans.xml",
  "/org/sgodden/tom/model/beans.xml") )
abstract class AbstractIntegrationTest extends AbstractTestNGSpringContextTests {
  @BeforeSuite def beforeSuite: Unit = {
    Logger.getLogger("org").setLevel(Level.INFO)
  }
}
